05 - HTML/README.md

```diff
@@ -1 +1,437 @@
+    </tr>
+</table>
+
+You can also merge columns or rows using the `colspan` and `rowspan` attributes:
+
+```
+<table>
+    <tr>
+        <td>A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td colspan="2">C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+```
+<table>
+    <tr>
+        <td>A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td colspan="2">C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+
+```
+<table>
+    <tr>
+        <td rowspan="2">A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td>C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+```
+
+<table>
+    <tr>
+        <td rowspan="2">A</td>
+        <td>B</td>
+    </tr>
+    <tr>
+        <td>C</td>
+    </tr>
+    <tr>
+        <td>D</td>
+        <td>E</td>
+    </tr>
+</table>
+
+## Forms
+
+Forms are one of the most important elements in web applications, these are the way that users can interact with the data, by creating, updating or even deleting data from their records. In social networks are the way you post status, upload your photos, create an account, do log in into the application, etc.
+
+Knowing only HTML will be useless to make forms to be working, this will only create a space to write information or hold a path for a file, do selections, etc.
+
+The main tag for this is `form` which will be wrapping every other element. We have the `input`, `textarea` and `select` tags here.
+
+The `input` tag is very versatile, we can change the way it will pass data using the `type` attribute.
+
+```
+<input type="text">
+<input type="password">
+<input type="file">
+<input type="checkbox">
+<input type="radio">
+<input type="submit">
+```
+
+We have also the `textarea` which is a similar element to `<input type="text">`.
+
+And finally we have the `select` which will contain a `option` for every option you can select from the list:
+
+```
+<select>
+    <option value="1">One</option>
+    <option value="2">Two</option>
+    <option value="3">Three</option>
+    <option value="4">Four</option>
+    <option value="5">Five</option>
+</select>
+```
+
+You see that `value` attribute there? That's the real value that will be sent, the text inside the `option` tag it's only a display value for the user.
+
+To finish with **tags** I will show you some that has no visual impact for website but are very useful:
+
+The `link` tag will help you to import external resources to your site. This will helping us to having more control and a readable code. This usually go inside the `head` tag and goes together with a `src` attribute. The value of this attribute will be pointing to the resource we want to include it.
+
+```
+<link href="resource.path">
+```
+
+Adding comments to the HTML code is useful to explain others what are you doing or probably explaining why you take that approach. To add comments you need the following syntax:
```

Comment:
`is useful to explain others what are you doing` 

could be?

`is useful to explain others what you are doing`